src/CbcCInterface.jl

```diff
@@ -57,6 +57,15 @@ export CbcModel,
     secondaryStatus,
     addSOS
 
+#The follow unsafe methods are created but not exported
```

Comment:
We don't list unexported functions, so it's inconsistent to list these here. Please remove.
See also the new guidance on naming unexported functions: http://www.juliaopt.org/JuMP.jl/dev/style/#Exported-and-non-exported-names-1. That's not necessary to address right now.

Reply:
So should I export them or make them underscored?

Reply:
Just removing this comment is fine for now (to make it consistent with the rest of the code). Eventually when this package is updated to follow the new conventions, the functions should have underscores.

Reply:
As they are being used outside of the CbcCInterface module I opted to export them